src/ebur128.rs

```diff
@@ -2018,13 +2018,13 @@ mod tests {
                 ebu.true_peak(c).unwrap(),
                 ebu_c.true_peak(c).unwrap(),
                 // For a performance-boost, filter is defined as f32, causing slightly lower precision
-                abs <= 0.000002,
+                abs <= 0.000004,
```

Comment:
I think at some point we should simply drop the C tests as long as we have new tests that cover all the components equally well. The implementation is diverging a lot from the C implementation already and at some point keeping the outputs similar enough with all the floating point inaccuracies depending on operation order/etc this is going to be annoying to maintain.

Not necessary as part of this PR, just mentioning this :)

Reply:
https://github.com/sdroege/ebur128/issues/37